sim/paladin/holy_talents.go

```diff
@@ -0,0 +1,100 @@
+package paladin
```

Comment:
Can we rename the talent files to `talents_spec.go` instead of `spec_talents.go` this would put them next to each other when sorting xD